### PR TITLE
Migrating the CNIL "accompagnement renforce" announcement from the HF blog

### PR DESCRIPTION
**Pull request by yjernite**
Copy-pasted from LinkedIn to make it easier to access!

Originally at:
https://www.linkedin.com/pulse/accompagnement-renforc%25C3%25A9-de-la-cnil-et-protection-des-donn%25C3%25A9es/

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -2232,3 +2232,11 @@
     - aws
     - partnership
     - guide
+
+- local: cnil-accompaniment
```
Comment (julien-c) [inline _blog.yml:2236]:
```suggestion
- local: cnil
```

either just `cnil` or a more general `data-protection-at-hf` or `eu-data-protection`?

(accompaniement seems off to me)

File: cnil-accompaniment.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Hugging Face Selected for the French Data Protection Agency Enhanced Support Program"
+thumbnail: /blog/assets/146_cnil-accompaniment/logo.png
+authors:
+- user: yjernite
+- user: julien-c
+- user: annatrdj
```
Comment (julien-c) [inline cnil-accompaniment.md:7]: also @ImaBello!! as she'll be main point of contact :)

**Review by julien-c — APPROVED** [review]
left small suggestions but lgtm

File: cnil-accompaniment.md (hunk 2 above)
Comment (yjernite) [reply cnil-accompaniment.md]: done!

**Pull request merged**